public/editor-client/src/defaultTemplates/index.ts

```diff
@@ -149,46 +152,61 @@ const defaultPopups = (
 
 const defaultStories = (
   config: Config
-): DefaultTemplate<StoriesWithThumbs, BlocksArray<DefaultBlock>> => {
-  const { storiesUrl } = config.api.templates;
+): LayoutsDefaultTemplate<
+  StoriesWithThumbs,
+  BlocksArray<DefaultBlock>,
+  LayoutsPages
+> => {
+  const { storiesChunkUrl, storiesDataUrl, storiesPagesUrl } =
+    config.api.templates;
+  const { templatesImageUrl } = config.api;
 
   return {
     async getMeta(res, rej) {
       try {
-        const meta: Stories = await fetch(`${storiesUrl}/meta.json`).then((r) =>
-          r.json()
+        const { templates, categories } = await getDefaultStories(
+          storiesChunkUrl
         );
 
         const data = {
-          ...meta,
-          stories: meta.stories.map((story) => {
-            return {
-              ...story,
-              thumbnailSrc: `${storiesUrl}/thumbs/${story.pages[0].id}.jpg`,
-              pages: story.pages.map((page) => {
-                return {
-                  ...page,
-                  thumbnailSrc: `${storiesUrl}/thumbs/${page.id}.jpg`
-                };
-              })
-            };
-          })
+          stories: convertStories(templates, `${templatesImageUrl}`),
```

Comment:
here is the same story with `${templatesImageUrl}`

should be used simple: stories: `convertStories(templates, templatesImageUrl)`